File: app/src/main/java/com/lkms/data/model/java/Item.java
package com.lkms.data.model.java;

import com.google.gson.annotations.SerializedName;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

/**
 * Đại diện cho một hàng trong bảng "Item".
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Item {

    @SerializedName("itemId")
    private Integer itemId;

    @SerializedName("itemName")
    private String itemName;

    @SerializedName("casNumber")
    private String casNumber;

    @SerializedName("lotNumber")
    private String lotNumber;

    @SerializedName("quantity")
    private Integer quantity;

    @SerializedName("unit")
    private String unit;

    @SerializedName("location")
    private String location;

    @SerializedName("expirationDate")
    private String expirationDate;
}

File: app/src/main/java/com/lkms/data/model/Item.kt
// Trong file: data/model/Item.kt
package com.lkms.data.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/**
 * Đại diện cho một hàng trong bảng "Item".
 */
@Serializable
data class Item(
    @SerialName("itemId")
    val itemId: Int,

    @SerialName("itemName")
    val itemName: String? = null,

    @SerialName("casNumber")
    val casNumber: String? = null,

    @SerialName("lotNumber")
    val lotNumber: String? = null,

    @SerialName("quantity")
    val quantity: Int? = null,

    @SerialName("unit")
    val unit: String? = null,

    @SerialName("location")
    val location: String? = null,

    @SerialName("expirationDate")
    val expirationDate: String? = null
)